src/main/java/domain/Crews.java

```diff
@@ -2,67 +2,81 @@
+        Crew crew = findCrewByUsername(username);
+        return crew.getAttendanceAnalyze(standard);
     }
 
-    public AttendanceRecord getAttendanceAllResult(String username, LocalDateTime localDateTime) {
-        Crew findCrew = findCrew(username);
-        return findCrew.getAttendanceAllResult(localDateTime);
+    public Map<Username, AttendanceAnalyze> findExpulsionCrews(LocalDate standard) {
+        List<Crew> expulsionCrews = getExpulsionCrewsWithOrders(standard);
+        return createExpulsionCrewsInfo(standard, expulsionCrews);
     }
 
-    public Map<Crew, AttendanceRecord> getHighAbsenceLevelCrewsInfo(LocalDateTime localDateTime) {
-        List<Crew> highAbsenceLevelCrews = crews.stream()
-                .filter(crew -> crew.isHighAbsenceLevel(localDateTime))
-                .toList();
-        Map<Crew, AttendanceRecord> result = new TreeMap<>();
-        highAbsenceLevelCrews.forEach(crew -> {
-            result.put(crew, crew.getAttendanceAllResult(localDateTime));
-        });
-        return result;
+    private Crew findCrewByUsername(String username) {
+        return crews.stream().filter(crew -> crew.isSameName(username))
+                .findAny()
+                .orElseThrow(() -> new IllegalArgumentException("[ERROR] 등록되지 않은 닉네임입니다."));
     }
 
-    private Crew findCrew(String username) {
-        return crews.stream()
-                .filter(crew -> crew.getUserName().equals(username)).findAny()
-                .orElseThrow(() -> new IllegalArgumentException("[ERROR] 존재하지 않은 크루입니다."));
+    private LinkedHashMap<Username, AttendanceAnalyze> createExpulsionCrewsInfo(LocalDate standard,
+                                                                                List<Crew> expulsionCrews) {
+        return expulsionCrews.stream().collect(Collectors.toMap(
+                Crew::getUsername,
+                crew -> crew.getAttendanceAnalyze(standard),
+                (existing, replacement) -> existing,
+                LinkedHashMap::new
+        ));
+    }
+
+    private List<Crew> getExpulsionCrewsWithOrders(LocalDate standard) {
+        return crews.stream().filter(crew -> crew.isExpulsionTarget(standard))
+                .sorted(new Comparator<Crew>() {
+                    @Override
+                    public int compare(Crew o1, Crew o2) {
+                        AttendanceAnalyze attendanceAnalyzeO1 = o1.getAttendanceAnalyze(standard);
+                        AttendanceAnalyze attendanceAnalyzeO2 = o2.getAttendanceAnalyze(standard);
+                        int compare = Integer.compare(attendanceAnalyzeO2.calculatePenaltyPoints(),
+                                attendanceAnalyzeO1.calculatePenaltyPoints());
+                        if (compare == 0) {
+                            return o1.getUsername().compareTo(o2.getUsername());
+                        }
+                        return compare;
+                    }
+                })
+                .toList();
```

Comment:
Comparator는 함수형 인터페이스니까 익명 클래스 방식 대신에 람다식을 활용해도 될 것 같네요. 😃 

```suggestion
        return crews.stream().filter(crew -> crew.isExpulsionTarget(standard))
                .sorted((o1, o2) -> {
                    AttendanceAnalyze attendanceAnalyzeO1 = o1.getAttendanceAnalyze(standard);
                    AttendanceAnalyze attendanceAnalyzeO2 = o2.getAttendanceAnalyze(standard);
                    int compare = Integer.compare(attendanceAnalyzeO2.calculatePenaltyPoints(),
                            attendanceAnalyzeO1.calculatePenaltyPoints());
                    if (compare == 0) {
                        return o1.getUsername().compareTo(o2.getUsername());
                    }
                    return compare;
                })
                .toList();
```